site/docs/master/restic.md

```diff
@@ -307,14 +307,12 @@ should be taken (`backup.velero.io/backup-volumes`)
     - finds the pod volume's subdirectory within the above volume
     - runs `restic backup`
     - updates the status of the custom resource to `Completed` or `Failed`
-1. As each `PodVolumeBackup` finishes, the main Velero process captures its restic snapshot ID and adds it as an annotation
-to the copy of the pod JSON that's stored in the Velero backup. This will be used for restores, as seen in the next section.
+1. As each `PodVolumeBackup` finishes, the main Velero process adds it to the Velero backup in a file named `<backup-name>-podvolumebackups.json.gz`. This file gets uploaded to object storage alongside the backup tarball. It will be used for restores, as seen in the next section.
 
 ### Restore
 
-1. The main Velero restore process checks each pod that it's restoring for annotations specifying a restic backup
-exists for a volume in the pod (`snapshot.velero.io/<volume-name>`)
-1. When found, Velero first ensures a restic repository exists for the pod's namespace, by:
+1. The main Velero restore process checks each pod that it's restoring from a previously created restic backup by iterating through the contents of the `<backup-name>-podvolumebackups.json.gz` file.
```

Comment:
I think Steve already commented on this, but is this documentation accurate?

My expectation would be that the backup sync controller would take the contents of `<backup-name>-podvolumebackups.json.gz` and submit each item to the Kubernetes API server so that they exist in the cluster.

Reply:
I just finished doing the sync so yes, what you say is correct. But doesn't the restore need to do the same?

Reply:
I don't believe so. Once the `PodVolumeBackup`s exist in the Kubernetes API server from the sync, the restore process could query the Kubernetes API server for them, no need to download the gzipped json file. At least, that's how I'm reading the description in #1169.

So https://github.com/heptio/velero/blob/master/pkg/restic/restorer.go#L89 would be replaced (eventually) by a query to the Kubernetes API server for all `PodVolumeBackup`s labeled with the `Backup` name (can be gotten from the `Restore`).

Reply:
Ah. I see. Thanks for clarifying that. I'll fix this doc too.